examples/k3s.yaml

```diff
@@ -12,11 +12,11 @@
 # This example requires Lima v0.7.0 or later.
 
 images:
-  # Hint: run `limactl prune` to invalidate the "current" cache
-  - location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-amd64.img"
-    arch: "x86_64"
-  - location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-arm64.img"
-    arch: "aarch64"
+# Hint: run `limactl prune` to invalidate the "current" cache
+- location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-arm64.img"
+  arch: "aarch64"
```

Comment:
This indentation doesn't seem consistent with other files in the same directory

Reply:
Is is consistent with `alpine.yaml`, and is using the "proper" indentation of sequences in YAML. I would like to eventually convert the other files to match (I asked about this earlier in this "thread").

Reply:
Seems we have to fix `default.yaml`.

Can we automatically detect this kind of issues?

Reply:
> Can we automatically detect this kind of issues?

Yes, in `.yamllint` we have to set `indent-sequences` to `false` instead of `consistent` (the consistency check only works within a file, not across files).

The changes will have to be made manually though, as all the reformatters I'm aware of will drop comments. Also in `default.yaml` much of the sequences are commented out, so wouldn't be reformatted automatically anyways.

I can volunteer to make those edits (and change `indent-sequences`) in a follow-up PR, if we agree to do them. Unless @afbjorklund wants to make them as part of this PR, of course.

Reply:
I thought this would be a simple check :-) I'm OK with either one-standard-per-file, or wait until a global one emerges

So far I have found that `strictyaml` preserves comments, but it also adds drops the quotes from all string values...

Reply:
> So far I have found that `strictyaml` preserves comments, but it also adds drops the quotes from all string values...

I would want to test this. `ruamel.yaml` also claimed to retain comments, but broke badly when I tried it. And I think `strictyaml` may be based on `ruamel`, but I didn't look too closely.

> https://hitchdev.com/strictyaml/why/implicit-typing-removed/

I'm somewhat sympathetic with this issue, but the "Norway problem" has been fixed by the YAML 1.2 spec in 2009...
 
> I think indenting them is consistent with machine-readable "markup" (like XML), and not indenting them is consistent with human-readable "language" (like MD). Will probably be hard to keep everyone happy ?

Yes, that sounds right to me. I think the double-indented form comes from machine-generated YAML because it can be produced by recursive tree walking without the writer having to be context aware. Similar to simplistic C code generation:

```c
if (condition)
    {
        statement-1;
        statement-2;
    }
```

But a human would normally write this as:

```c
if (condition) {
    statement-1;
    statement-2;
}
```

Our YAML files are supposed to be read by humans, so should be formatted for that purpose. Otherwise we could be using JSON instead and avoid all these issues (any valid JSON file is automatically also a valid YAML file).